Example/Tests/GINIPDFDocumentTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  GINIPDFDocumentTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 9/14/17.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+class GINIPDFDocumentTests: XCTestCase {
+  
+    let pdfNonRotated = GINIPDFDocumentTests.loadPDFDocument(withName: "testPDF")
+    
+    func testNumberPages() {
```

Comment:
Testing with a PDF with more than one page might be interesting as a regression test. If you can find a two page one, you should add another test to make sure the numberPages property really counts pages.

Reply:
Yes, that could be another test case! 👍